flowsa/data_source_scripts/stewiFBS.py

```diff
@@ -43,12 +43,8 @@ def stewicombo_to_sector(yaml_load, method, fbsconfigpath=None):
     :param fbsconfigpath, str, optional path to an FBS method outside flowsa repo
     :return: df, FBS format
     """
-
     import stewicombo
-    from flowsa.data_source_scripts.EPA_NEI import drop_GHGs
 
-    # determine if fxns specified in FBS method yaml
-    functions = yaml_load.get('functions', [])
```

Comment:
If I recall we had added this "functions" to allow for some customized additional steps for some stewi based FBS, that we ultimately didn't use.

Reply:
Makes sense. My thought is that we can still pull this code out (it's not used for anything in `flowsa` besides `drop_GHGs`), and then if customized additional steps are desired in the future we have some new tools that can be used to define those additional steps. (For instance, to arrive at very similar functionality to the old code, we could modify the custom `yaml` loader to allow a list of functions to be passed after the `!script_function:` tag.)